docker/step/set_up_env.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
You might want to run `pre-commit install` before `git commit`.  The `pre-commit` will check for the copyright message. Currently, this change cannot pass the CI.